pymeasure/instruments/hp/hp33120A.py

```diff
@@ -112,6 +112,54 @@ def __init__(self, adapter, name="Hewlett Packard 33120A Function Generator", **
         values=AMPLITUDE_UNITS,
         map_values=True
     )
+    burst_state = Instrument.control(
+        "BM:STATE?", "BM:STATE %d",
+        """Enable or disable burst modulation""",
+        validator=strict_discrete_set,
+        values=[0, 1],
+    )
+    burst_source = Instrument.control(
+        "BM:SOURCE?", "BM:SOURCE %s",
+        """Select an internal or external gate source for burst modulation""",
```

Comment:
A `control`'s docstring should start with "Control". Similarly, a `measurement`'s docstring should start with "Get" or "Measure" and a `setting`'s docstring with "Set".
Please adjust all your docstrings.

Maybe other properties do not adhere to these conventions, but new properties should follow them.